rfcs/text/0008/threat.yml

```
@@ -161,10 +161,10 @@
       Traffic Light Protocol sharing markings.
 
       Expected values are:
```

```suggestion
      Recommendedded values are:
```

> I think we should leave it flexible. MISP, as an example, is using all lower-case TLP. I would prefer to set the examples to align with the standard, but I wouldn't want an error or to reject based on case.

I'm also fine with the uppercase values being a "soft" recommendation.

Let's leave the examples as all-uppercase, but instead of `Expected`, how about rephrasing to `Recommended`?